tests/webtests/WebTestCase.php

```diff
@@ -88,6 +88,9 @@ class WebTestCase extends PantherTestCase
 
     protected DevelopmentIssueRepository $jiraIssueRepository;
 
+    private string $surfConextRepresentativeAttributeName = '';
+    private string $surfConextRepresentativeAttributeValue = '';
+
     public static function setUpBeforeClass(): void
     {
         exec('cd /var/www/html && composer dump-env test -q && chmod 777 /tmp/spdashboard-webtests.sqlite');
```

Comment:
This we can do better by using sqlite in memory, I have an example setup in my private repo how to do this. Not for now, but later on. Basically I used this https://www.sitepoint.com/quick-tip-testing-symfony-apps-with-a-disposable-database/

Reply:
In this case, for the simple fixtures, I find it more flexible to simply store json blobs in txt files. I am interested in your thoughts about optimising our set up. Which is really basic, and bare bones at this point.